tests/Api/FocusTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic, NP. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     MIT http://opensource.org/licenses/MIT
+ */
+
+namespace Mautic\Tests\Api;
+
+class FocusTest extends MauticApiTestCase
+{
+    protected $skipPayloadAssertion = array('htmlMode', 'properties');
+
+    public function setUp()
+    {
+        $this->api = $this->getContext('focus');
+        $this->testPayload = array(
+            'name' => 'test',
+            'type' => 'notice',
+            'website' => 'http://',
+            'style' => 'bar',
+            'htmlMode' => 1,
+            'html' => '<div><strong style="color:red">html mode enabled</strong></div>',
+            'css' => '.mf-bar-collapser {border-radius: 0 !important}',
```

Comment:
@kuzmany While running the final tests before the API Library 2.9.0 release, the `Mautic\Tests\Api\FocusTest::testBatchEndpoints` test fails on `The ["focus" => "css"] doesn't exist in the response`. When I look at the Focus entity class, it does not have the `css` property. Why is it here?